src/main/java/ted/task/TaskList.java

```diff
@@ -0,0 +1,147 @@
+package ted.task;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import ted.exception.TedException;
+
+/**
+ * Represents list of tasks. A <code>TaskList</code> object corresponds to the tasks saved
+ * to the bot by the user.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    /**
+     * Creates empty TaskList object.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Creates Tasklist object with pre-existing tasks.
+     *
+     * @param tasks arraylist of tasks to be inserted into TaskList object.
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Returns number of tasks in the TaskList.
+     *
+     * @return size of TaskList.
+     */
+    public int getSize() {
+        return tasks.size();
+    }
+
+    /**
+     * Returns String representation of the list of tasks in the TaskList object.
+     *
+     * @return String of task list.
+     */
+    public String list() {
+        String listOfTasks = "";
+        for (int i = 0; i < tasks.size(); i++) {
+            int bulletPoint = i + 1;
```

Comment:
Not sure I'd call this a bulletPoint since it seems to be the index number of the tasks